rest/src/main/java/com/netflix/conductor/rest/controllers/ApplicationExceptionMapper.java

```diff
@@ -53,16 +53,7 @@ public class ApplicationExceptionMapper {
         EXCEPTION_STATUS_MAP.put(NoResourceFoundException.class, HttpStatus.NOT_FOUND);
     }
 
-    @ExceptionHandler(ClientAbortException.class)
```

Comment:
Reverted my prev change - I believe we should rather ignore it from alerts than changing the error logs here or something.
Added test coverage for the existing exception handler.